### PR TITLE
Fix/Update Plugin Models

### PR DESCRIPTION
**Pull request by zeroadam**
Update models for use with Plugins/publishing

### CODE REVIEW

File: src/Plugins/PublishRequestModel.cs [hunk 1]
```diff
@@ -32,14 +32,14 @@ public enum PublishType
...
+        /// Contains a value that indicates publishing to DocBook.
         /// </summary>
-        DocBookPdf,
+        DocBook,
 
         /// <summary>
         /// Contains a value that indicates publishing to AntennaHouse PDF.
         /// </summary>
-        AntennaHousePdf
+        AntennaHousePdf,
```
Comment (vasont) [inline src/Plugins/PublishRequestModel.cs:42]: Ignore Stylecop on these. It's a bug I believe in their rules/parser.

File: src/Plugins/PublishResultModel.cs [hunk 2]
```diff
@@ -34,5 +34,10 @@ public class PublishResultModel
         /// Gets or sets a value containing errors that occurred when attempting to initiate publishing.
         /// </summary>
         public string PublishInitiateErrors { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value containing the full path of the published file.
+        /// </summary>
+        public string ResultFilePathName { get; set; }
```
Comment (vasont) [inline src/Plugins/PublishResultModel.cs:41]: Is it a path or name? Comment says path? ResultFilePath or ResultFileName makes more sense?

File: src/Plugins/PublishRequestModel.cs (hunk 1 above)
Comment (vasont) [inline src/Plugins/PublishRequestModel.cs:42]: Ignore Stylecop on these. It's a bug I believe in their rules/parser.

File: src/Plugins/PublishResultModel.cs (hunk 2 above)
Comment (vasont) [reply src/Plugins/PublishResultModel.cs:41]: Also, I do not suggest passing full drive paths in any model across the network. We took out a bunch of that in Inspire. Security concern giving away full drive/path/name structure over the wire.

Comment (zeroadam) [reply src/Plugins/PublishResultModel.cs:41]: I would be inclined to change it to ResultFilePath. This is used internally to move the file into place, but, it is then eventually used to pass the download url. The drive path never gets sent out over the wire

File: src/Plugins/PublishRequestModel.cs (hunk 1 above)
Comment (zeroadam) [reply src/Plugins/PublishRequestModel.cs:42]: will do

**Pull request merged**